revolv/administrator/views.py

```diff
@@ -29,6 +35,423 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class AdministratorAccountingView(UserDataMixin, TemplateView):
+    """View of the administrator accounting page.
+    """
+    template_name = 'administrator/accounting.html'
+
+    def clean_project_name(self, name):
+        name = name.replace(' ', '-').replace(';', 'sm').replace(':', 'co').replace("'", 'ap').replace('!', 'ex')
+        return name
+
+    def get_context_data(self, **kwargs):
+        context = super(AdministratorAccountingView, self).get_context_data(**kwargs)
+
+        context['project_names'] = [p.title for p in Project.objects.all()]
+        context['admin_repayments'] = AdminRepayment.objects.repayments()
+        context['admin_reinvestments'] = AdminReinvestment.objects.reinvestments()
+        context['donations'] = Payment.objects.donations()
+        context['jsonurl'] = unicode(str(reverse('administrator:accountingJSON')))
+
+        context['clean_project_names'] = {}
+        clean_project_names = context['clean_project_names']
+        for p in context['project_names']:
+            clean_project_names[self.clean_project_name(p)] = p
+
+        return context
+
+class AccountingJSONView(View):
+    """
+    A view that returns a JsonResponse of all relevant accounting data.
+    This is queried by an AJAX request when creating the Accounting page.
```

Comment:
Could we maybe talk here about how the accounting info is calculated and how the AdminAdjustments play into it?